### ISSUE TITLE
[Site Request] jmanga

### ISSUE THREAD

**Issue by ghost** [manga-download/hakuneko#7279]
**Name of the website**
jmanga

**Website urls (examples below)**
- Site: [https://jmanga.org/](https://jmanga.org/)
- Manga List: [https://jmanga.org/filter/?p=1](https://jmanga.org/filter/?p=1)
- Manga example: [https://jmanga.org/read/%E6%81%A5%E3%81%98%E3%82%89%E3%81%86%E5%90%9B%E3%81%8C%E8%A6%8B%E3%81%9F%E3%81%84%E3%82%93%E3%81%A0-raw/](https://jmanga.org/read/%E6%81%A5%E3%81%98%E3%82%89%E3%81%86%E5%90%9B%E3%81%8C%E8%A6%8B%E3%81%9F%E3%81%84%E3%82%93%E3%81%A0-raw/)
- Chapter online viewer example: [https://jmanga.org/read/%E6%81%A5%E3%81%98%E3%82%89%E3%81%86%E5%90%9B%E3%81%8C%E8%A6%8B%E3%81%9F%E3%81%84%E3%82%93%E3%81%A0/ja/chapter-1.1-raw/](https://jmanga.org/read/%E6%81%A5%E3%81%98%E3%82%89%E3%81%86%E5%90%9B%E3%81%8C%E8%A6%8B%E3%81%9F%E3%81%84%E3%82%93%E3%81%A0/ja/chapter-1.1-raw/)

**Languages**
japanese

**Website relationship*
japanese version of the already implemented [mangareader.to](mangareader.to). There are also a lot of manga (or at least the japanese versions of manga) not available on mangareader.to, like for example the example given under the "Manga example".

**Additional details**

**Comment by MikeZeDev** [comment]
Another day, another people who dont use search...

![image](https://github.com/user-attachments/assets/67fbb57b-fc54-4cd6-a763-c01ff4cf654a)

**Comment by ghost** [comment]
@MikeZeDev Yes, I've seen that issue. But for some reason, jmanga and rawotaku both neither show up in the source selector nor work with direct urls for me, so I thought that maybe there was a mistake. Could you maybe check if it works on your version? If so, then maybe I made a mistake with the download. 
(In case it helps, I'm using the 0.89.0 version of HaruNeko-nw i got from [here](hhttps://github.com/manga-download/haruneko/releases/tag/240708T0032) for the windows 64-bit system. I've also tried the old HakuNeko, but it doesn't work there either)

Here I attached a video to show it:
https://github.com/user-attachments/assets/55531450-449c-4f1d-af87-9c975e774551

**Comment by MikeZeDev** [comment]
In any case : dont open a "website suggestion" issue for a website when there is already an issue for it 🤷 .
My answer was still accurate (i didnt want to delete it  )

Open an untagged issue and ask your question. ;)

You problem is that you want a website that is coded but not available yet in Haruneko.

![image](https://github.com/user-attachments/assets/d191f0bd-ec83-42e4-831c-f6cafcd89d76)

**master** branch , from where you can download binaries, dont have JManga.

https://github.com/manga-download/haruneko/blob/master/web/src/engine/websites/JManga.ts

However, website is ready.

https://github.com/manga-download/haruneko/blob/liliana/web/src/engine/websites/JManga.ts

In order to include this code into master, this pull request https://github.com/manga-download/haruneko/pull/455/files have to be "MERGED", meaning it needs to be reviewed by a peer first.

TLDR : wait, just wait.
